### PR TITLE
plugins: avoid `csgrep: Argument list too long` error

### PR DESCRIPTION
**Pull request by kdudka**
... on glob expansion in filter_hook() and make sure we have read
permission on the capture files in case they were created with too
restrictive umask.

### CODE REVIEW

File: py/plugins/cbmc.py [hunk 1]
```diff
@@ -119,13 +119,19 @@ def create_cap_dir_hook(results, mock):
...
                   set -ex
-                  for file in %s/pid-*.out; do
+                  cd '%s'
+                  for file in pid-*.out; do
                       cbmc-convert-output -a < \"$file\" > \"$file.conv\"
                   done
-                  csgrep --mode=json --remove-duplicates '%s'/pid-*.out.conv > '%s'
-                  """ % (src_dir, src_dir, dst)
+                  csgrep --mode=json --remove-duplicates pid-*.out.conv > '%s'
+                  """ % (src_dir, dst)
```
Comment (lzaoral) [inline py/plugins/cbmc.py:135]: Could you use `f-strings` for consistency with the rest?

**Review by lzaoral — APPROVED** [review]
Thanks, @kdudka! LGTM (except for a small nitpick).

File: py/plugins/cbmc.py (hunk 1 above)
Comment (kdudka) [reply py/plugins/cbmc.py:135]: I was not sure whether they work with the multi-line string literals but they do :-)

**Comment by kdudka** [comment]
Thanks for review!  Merging...

**Pull request closed without merging**